ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -3369,6 +3369,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi-compute-gpu
+- as: nutanix-ipi-multi-nics-f28
```

Comment:
can we add some upgrade ci jobs for the install profile ?

Reply:
BTW, can we change `multi-nics` to `multi-nic` to keep consistent with vsphere case.

Reply:
Since it's TP feature and FEATURE_SET: CustomNoUpgrade, do we need to add upgrade job? Or we can add it after GA?

Reply:
ah, I forgot that, then we do not need to add upgrade job.